docs/interactions/Slash_Commands.md

```diff
@@ -742,6 +782,31 @@ Edits a followup message for an Interaction. Functions the same as [Edit Webhook
 
 Deletes a followup message for an Interaction. Returns `204` on success.
 
+## Get Guild Application Command Permissions % GET /applications/{application.id#DOCS_TOPICS_OAUTH2/application-object}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands/permissions
+
+Fetches command permissions for all commands for your application in a guild. Returns an array of [ApplicationCommandPermissions](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommandpermissions).
+
+
+## Get Application Command Permissions % GET /applications/{application.id#DOCS_TOPICS_OAUTH2/application-object}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands/{command.id#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommand}/permissions
+
+Fetches command permissions for a specific command for your application in a guild. Returns an array of [ApplicationCommandPermissions](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommandpermissions).
```

Comment:
Seems this endpoint returns more than just an array of ApplicationCommandPermissions from what I can see

![image](https://user-images.githubusercontent.com/29981918/112572354-ed1a1680-8de1-11eb-9b38-67ebee23938a.png)

Reply:
see up https://github.com/discord/discord-api-docs/pull/2737#pullrequestreview-621697678

Reply:
> see up [#2737 (review)](https://github.com/discord/discord-api-docs/pull/2737#pullrequestreview-621697678)

Oh, didn't see that as I had files changed open while implementing it, not conversation